saleor/attribute/models/base.py

```diff
@@ -319,10 +319,18 @@ def bulk_update_or_create(self, objects_data):
             )
 
         if objects_to_be_updated:
-            self.bulk_update(
-                objects_to_be_updated,
-                fields=update_fields,
-            )
+            from ..lock_objects import attribute_value_qs_select_for_update
```

Comment:
If you move `AttributeValueManager` to `managers.py`, you could avoid circular imports. We have such file for products: https://github.com/saleor/saleor/blob/main/saleor/product/managers.py